README.md

```diff
@@ -101,10 +234,32 @@ Use the client ID from yout app listing when initializing your MSALPublicClientA
     }
 ```
 
+### Silently Acquiring an Updated Token (Swift)
+```swift
+
+    application.acquireTokenSilent(forScopes: kScopes, user: user) { (result, error) in
+    
+        if !(error != nil)  {
+
+          accessToken = result.accessToken!
+                        
+                        
+         } else  {
+                        
+          // Interactive auth will be required
```

Comment:
that's actually wrong, right? We need to specifically look for interaction_required error rather than a generic error case, otherwise we'll fall back to AT for net failures.